<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="marco-TP1" failures="1" tests="1" name="Command line test" time="0.075" errors="0" timestamp="2021-09-16T02:13:21 PET">
  <testcase classname="cs.ing.software.lab1.PruebaTest" name="Test01" time="0.019">
    <failure type="java.lang.AssertionError" message="expected [Automovil no se puede reabastecer el tanque, esta lleno] but found [Camion viajo 4.0 km y aun tiene 6.40 de combustible]">
      <![CDATA[java.lang.AssertionError: expected [Automovil no se puede reabastecer el tanque, esta lleno] but found [Camion viajo 4.0 km y aun tiene 6.40 de combustible]
at cs.ing.software.lab1.PruebaTest.Test01(PruebaTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- Test01 -->
</testsuite> <!-- Command line test -->
